services/db/AxiosInstance.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosRequestConfig, AxiosResponse } from "axios"
+
+import logger from "@logger/logger"
+
+const axios = require("axios")
+
+// Env vars
+const { GITHUB_ORG_NAME } = process.env
+
+const requestFormatter = (config: AxiosRequestConfig) => {
+  logger.info("Making GitHub API call")
+  return {
+    ...config,
+    headers: {
+      ...config.headers,
+      "Content-Type": "application/json",
+    },
+  }
+}
+
+const respHandler = (response: AxiosResponse) => {
+  // Any status code that lie within the range of 2xx will cause this function to trigger
+  const remainingRequests = response.headers["x-ratelimit-remaining"]
+  if (remainingRequests < 2000) {
```

Comment:
nit: would prefer to have a constant specifying the limit (eg: `GITHUB_API_LIMIT = 2500`) and specifying this in terms of the limit (`remainingRequests < GITHUB_API_LIMIT * 0.6`) for clarity

Reply:
Makes sense, https://github.com/isomerpages/isomercms-backend/pull/440/commits/ea6d2872ca8347fd0086bb77dc8e7f6a35cc9aa8